command/transfer.go

```diff
@@ -1,10 +1,47 @@
 package command
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/rubixchain/rubixgoplatform/core/model"
 )
 
 func (cmd *Command) TransferRBT() {
+	if cmd.senderAddr == "" {
+		cmd.log.Info("Sender address cannot be empty")
+		fmt.Print("Enter Sender DID : ")
+		_, err := fmt.Scan(&cmd.senderAddr)
+		if err != nil {
+			cmd.log.Error("Failed to get Sender DID")
+			return
+		}
+	}
+	if cmd.receiverAddr == "" {
+		cmd.log.Info("Receiver address cannot be empty")
+		fmt.Print("Enter Receiver DID : ")
+		_, err := fmt.Scan(&cmd.receiverAddr)
+		if err != nil {
+			cmd.log.Error("Failed to get Receiver DID")
+			return
+		}
+	}
+	if strings.Contains(cmd.senderAddr, ".") || strings.Contains(cmd.receiverAddr, ".") {
```

Comment:
Not necessarily just `.` . The better check will be to make sure string is alphanumeric.